### ISSUE TITLE
[BUG]: Crash when selecting export from main menu if field not selected

### ISSUE THREAD

**Issue by jmh579** [PhenoApps/Field-Book#977]
### Description

Version 5.6.19 on Boox Palma

When Field Book is restarted, no field is selected by default. If no field is manually selected before attempting export, Field Book crashes when the export option is selected from the main menu. If the app is restarted and a field is selected, the export works as expected.

### Expected behavior

A default field should be loaded when the app starts, preferably the last one used.

### Field Book version

5.6.19

**Comment by trife** [comment]
```
      Fatal Exception: java.lang.NullPointerException: Attempt to invoke virtual method 'java.lang.String com.fieldbook.tracker.objects.FieldObject.getExp_name()' on a null object reference
       at com.fieldbook.tracker.utilities.ExportUtil.exportLocal(ExportUtil.kt:227)
       at com.fieldbook.tracker.utilities.ExportUtil.exportPermission(ExportUtil.kt:118)
       at com.fieldbook.tracker.utilities.ExportUtil.export(ExportUtil.kt:95)
       at com.fieldbook.tracker.utilities.ExportUtil.exportActiveField(ExportUtil.kt:82)
       at com.fieldbook.tracker.activities.ConfigActivity.lambda$loadScreen$2$com-fieldbook-tracker-activities-ConfigActivity(ConfigActivity.java:340)
       at com.fieldbook.tracker.activities.ConfigActivity$$ExternalSyntheticLambda2.onItemClick(:6)
       at android.widget.AdapterView.performItemClick(AdapterView.java:330)
       at android.widget.AbsListView.performItemClick(AbsListView.java:1215)
       at android.widget.AbsListView$PerformClick.run(AbsListView.java:3207)
       at android.os.Handler.handleCallback(Handler.java:938)
       at android.os.Handler.dispatchMessage(Handler.java:99)
       at android.os.Looper.loop(Looper.java:223)
       at android.app.ActivityThread.main(ActivityThread.java:7811)
       at java.lang.reflect.Method.invoke(Method.java)
       at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:592)
       at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:952)
    
```

**Comment by chaneylc** [comment]
This seems to be fixed on 5.6.20.

The only way I could have existing fields but none selected was importing a database. In this case when I tested export or collect I get a toast saying, "No field is currently loaded...".
